### ISSUE TITLE
Invalid depth level: nil

### ISSUE THREAD

**Issue by gligoran** [markets/jekyll-timeago#13]
Hey,

I'm getting this exception:

```Liquid Exception: Invalid depth level: nil in about.md```

My `about.md` looks like this:

```
---
---

{% timeago 2000-1-1 %}
```

I very unfamiliar with ruby, but after some playing with [`filter.rb`](https://github.com/markets/jekyll-timeago/blob/master/lib/jekyll-timeago/filter.rb), it seems that `options` are not set because `setup` wasn't called.

I'm using Jekyll 3.0.3 and Ruby 2.2.4p240.

**Comment by markets** [comment]
Hi @gligoran, thanks for reporting!

Yes, you're right, last released version (v0.8.2) doesn't work with Jekyll 3. I'm working on this here: https://github.com/markets/jekyll-timeago/pull/12. It's pretty much done, just some docs and internals are pending and I'll ready to push a new version to RubyGems. 

Meanwhile, you can use this branch by:

```
gem 'jekyll-timeago', github: 'markets/jekyll-timeago', branch: 'improve_ci'
```

It must work, all tests are passing according to Travis: https://travis-ci.org/markets/jekyll-timeago/builds/104320132

**Comment by markets** [comment]
hey @gligoran I finally finished that work and now this plugin is compatible with all Jekyll versions (v1, v2 and v3). 

I just pushed a new version to RubyGems: https://rubygems.org/gems/jekyll-timeago/versions/0.9.0.
